From 27a08856a62b32d028315951da06bf091bd22bf6 Mon Sep 17 00:00:00 2001
From: Bill Hoffa <wghoffa@us.ibm.com>
Date: Thu, 26 Feb 2015 12:50:20 -0600
Subject: [PATCH 1/5] Provide OCC Target with FRU_ID attribute

Required with github commits:
https://github.com/open-power/habanero-xml/commit/dc738f40fc6e6c63c2bf6e25dca0f85987c870b8
and
https://github.com/open-power/palmetto-xml/commit/84f5bf5cc10459a85340cd0cc82a8c65b5bc4c97

Change-Id: I8dd08fe5b4649f1398675c47db3abbdb8a5363b0
RTC:124817
(cherry picked from commit 15d270e5f3f7b49fd26e2656b6d5b617aea207d9)
---
 src/usr/targeting/common/xmltohb/target_types.xml |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/usr/targeting/common/xmltohb/target_types.xml b/src/usr/targeting/common/xmltohb/target_types.xml
index 4dcc796..6dc4e58 100644
--- a/src/usr/targeting/common/xmltohb/target_types.xml
+++ b/src/usr/targeting/common/xmltohb/target_types.xml
@@ -1726,6 +1726,9 @@
     <attribute>
         <id>OCC_MASTER_CAPABLE</id>
     </attribute>
+    <attribute>
+        <id>FRU_ID</id>
+    </attribute>
 </targetType>
 
 </attributes>
-- 
1.7.4.1

